Data/TOR/Story/XML/10540.xml

```diff
@@ -599,63 +599,63 @@ anything of interest on my end.</EnglishText>
       <PointerOffset>2226</PointerOffset>
       <JapaneseText>ワン・ギン様のところの
 お迎えをやってるって？</JapaneseText>
-      <EnglishText/>
+      <EnglishText>You were going to pick it up at Wang Gin's place?</EnglishText>
       <Notes/>
       <SpeakerId>9</SpeakerId>
       <Id>36</Id>
-      <Status>To Do</Status>
+      <Status>Editing</Status>
     </Entry>
     <Entry>
       <PointerOffset>2248</PointerOffset>
       <JapaneseText>あらあら……
 大変ねえ……</JapaneseText>
-      <EnglishText/>
+      <EnglishText>Oh my goodness...</EnglishText>
       <Notes/>
       <SpeakerId>9</SpeakerId>
       <Id>37</Id>
-      <Status>To Do</Status>
+      <Status>Editing</Status>
     </Entry>
     <Entry>
       <PointerOffset>2298</PointerOffset>
       <JapaneseText>盗賊が……!?
 よかったわ……
 うちは襲われなくて……</JapaneseText>
-      <EnglishText/>
+      <EnglishText>Bandits?! I'm glad they didn't get to this place.</EnglishText>
       <Notes/>
       <SpeakerId>9</SpeakerId>
       <Id>38</Id>
-      <Status>To Do</Status>
+      <Status>Editing</Status>
     </Entry>
     <Entry>
       <PointerOffset>2320</PointerOffset>
       <JapaneseText>店の改装資金の
 ２０万ガルドが手元にあるのよー。</JapaneseText>
-      <EnglishText/>
+      <EnglishText>I've got 200,000 Gald on hand to renovate the inn!</EnglishText>
       <Notes/>
       <SpeakerId>9</SpeakerId>
       <Id>39</Id>
-      <Status>To Do</Status>
+      <Status>Editing</Status>
     </Entry>
     <Entry>
       <PointerOffset>2359</PointerOffset>
       <JapaneseText>天変地異の前触れ？
 商売やってるどころじゃないわ……</JapaneseText>
-      <EnglishText/>
+      <EnglishText>A warning of a disaster? I can't be doing business at this hour.</EnglishText>
```

Comment:
I believe context is when the earthquake happens.
Perhaps something along the lines of "Could this earthquake be heralding a disaster? I can't be doing business at a time like this..."